src/KNNClassifier/index.js

```diff
@@ -72,7 +87,7 @@ class KNN {
     }
   }
 
-  clearClass(classIndexOrLabel) {
+  clearLabel(classIndexOrLabel) {
     let classIndex;
```

Comment:
This doesn't matter so much b/c it's internal but we could change to `labelIndex`?

Reply:
yes! Just changed it.